### PR TITLE
Add entries to LevelTable

### PR DESCRIPTION
**Pull request by ShammyLevva**
Interpolated values based on crafted gear for those levels, appears consistent with previous levels.

### CODE REVIEW

**Comment by ShammyLevva** [comment]
Add preparatoryTouch proper graphics icon

File: app/js/ffxivcraftmodel.js [hunk 1]
```diff
@@ -1946,7 +1946,17 @@ var LevelTable = {
     67: 282,
     68: 285,
     69: 288,
-    70: 290 
+    70: 290,
+    71= 330,
```
[the remark this answers is not in the document]
Comment (ShammyLevva) [reply app/js/ffxivcraftmodel.js]: Sorry I could have sworn I did a search n replace as I saw : was required.

**Comment by ShammyLevva** [comment]
The conflicting image file replaces the ? with the correct icon.

**Comment by ShammyLevva** [comment]
Later commits today add framework for displaying 100% efficiency scores. However I've not dug deep enough into code to determine where the values should be set so placeholders at present.

**Comment by doxxx** [comment]
You'll need to rebase onto the latest master and resolve the conflicts. When resolving the `app/locale/fr.json` conflict, take the version from master rather than your branch.

**Comment by doxxx** [comment]
I was planning on using a different method for this, so if you don't mind I'll steal your level table entries and re-implement the rest.

**Comment by ShammyLevva** [comment]
Yeah that's fine. what exactly do you need me to change?

**Comment by doxxx** [comment]
Sorry, I wasn't clear. I want to split this into two PRs:

1. All the changes needed to visualize the 100% efficiency progress and quality values (even if they're incorrect). I'll do this one myself.
2. All the model changes needed to simulate levels 71-80 better.

I should have the first one done later tonight.

**Pull request closed without merging**